### PR TITLE
finish step 1: About The Team

### LINKED ISSUE #1: about the team
## 1. about the team

> assigned to Evan

__As a site visitor, I want to know about your team and who is in it__

### Repo

- [x] developed on a branch named `about-the-team`

### README.md

- [x] create the file
- [x] the group name as a main title
- [x] a paragraph introduction to the team
- [x] a list containing each team member's name (not links, just text)

---

### PR DESCRIPTION
**Pull request by colevandersWands**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# superHeros
\ No newline at end of file
+# Super Heros
+
+Group 6 in class 11/12.
```
Comment (maniiasov) [inline README.md:3]: Group 5

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+- Laura
+- Solomon
+- Raoul
+- Chyngyz
+- [Evan](https://badgerbadgerbadgerbadger.com)
```
Comment (maniiasov) [inline README.md:9]: Just name without link

Comment (colevandersWands) [reply README.md]: Fixed in commit  c86dbf3

**Comment by maniiasov** [comment]
Perfect

File: README.md (hunk 1 above)
Comment (colevandersWands) [reply README.md]: @maniiasov , is this correct? 
9018da9

**Review by lauraramirez220612 — COMMENTED** [review]
Can you write in Bold

**Pull request merged**